R/get_genstate_vax.R

```diff
@@ -0,0 +1,46 @@
+#' Get general population state COVID vaccine data from CDC and Census 2020 18+ pop estimates
+#'
+#' @description Pulls the latest data from the CDC on COVID vaccinations initiated
+#' for adults in a state and then merges alongside a population number from the 18+ 2020
+#' Census population estimates for that state to use for rate calculation.
```

Comment:
Realllyyy little thing, but thoughts on adding something to the docs saying this is for adults 18+? Feels like something I will personally forget at some point 😭